app/org/sagebionetworks/bridge/services/SurveyService.java

```diff
@@ -113,45 +163,54 @@ public Survey publishSurvey(StudyIdentifier study, GuidCreatedOnVersionHolder ke
     public Survey versionSurvey(GuidCreatedOnVersionHolder keys) {
         checkArgument(StringUtils.isNotBlank(keys.getGuid()), "Survey GUID cannot be null/blank");
         checkArgument(keys.getCreatedOn() != 0L, "Survey createdOn timestamp cannot be 0");
-        
+
         return surveyDao.versionSurvey(keys);
     }
-    
+
     /**
-     * Delete this survey. Survey still exists in system and can be retrieved by
-     * direct reference (URLs that directly reference the GUID and createdOn
-     * timestamp of the survey), put cannot be retrieved in any list of surveys,
-     * and is no longer considered when finding the most recently published
-     * version of the survey.
-     * 
-     * @param keys
+     * Logically delete this survey (mark it deleted and do not return it from any list-based APIs; continue 
+     * to provide it when the version is specifically referenced). Once a survey is published, you cannot 
+     * delete it, because we do not know if it has already been dereferenced in scheduled activities. Any 
+     * survey version that could have been sent to users will remain in the API so you can look at its 
+     * schema, etc. This is how study developers should delete surveys. 
      */
-    public void deleteSurvey(GuidCreatedOnVersionHolder keys) {
+    public void deleteSurvey(StudyIdentifier studyId, GuidCreatedOnVersionHolder keys) {
```

Comment:
Minor: Do we need to take in the studyId here? Is it just for consistency with deleteSurveyPermanently?